src/collections/operations/findByID.ts

```diff
@@ -67,22 +66,25 @@ async function findByID<T extends TypeWithID>(
   // If errors are disabled, and access returns false, return null
   if (accessResult === false) return null;
 
-  const query = await Model.buildQuery({
-    where: combineQueries({ _id: { equals: id } }, accessResult),
-    payload,
+
+  const findArgs: FindArgs = {
+    collection: collectionConfig.slug,
+    where: combineQueries({ id: { equals: id } }, accessResult),
     locale: req.locale,
-  });
+    limit: 1,
+  };
 
   // /////////////////////////////////////
   // Find by ID
   // /////////////////////////////////////
 
-  if (!query.$and[0]._id) throw new NotFound(t);
+  if (!findArgs.where.and[0].id) throw new NotFound(t);
 
   if (!req.findByID) req.findByID = {};
 
   if (!req.findByID[collectionConfig.slug]) {
-    const nonMemoizedFindByID = async (q) => Model.findOne(q, {}).lean();
+    const nonMemoizedFindByID = async (q: FindArgs) => (await req.payload.db.find(q)).docs[0];
```

Comment:
I will always call out single character variable names when I do my code review. I would prefer `query` or `args` or anything over `q` here, though it is pedantic and I can get over it since it is only used in this line.

Reply:
Don't worry, I don't want to trigger your single-character-variable-names OCD. I've changed it to query :p